src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/KeyInfoX509Data.cs

```diff
@@ -301,5 +301,543 @@ public override void LoadXml(XmlElement element)
                 AddCertificate(new X509Certificate2(Convert.FromBase64String(Utils.DiscardWhiteSpaces(node.InnerText))));
             }
         }
+
+        //
+        // This is a pretty "crude" implementation of BigInt arithmetic operations.
+        // This class is used in particular to convert certificate serial numbers from
+        // hexadecimal representation to decimal format and vice versa.
+        //
+        // We are not very concerned about the perf characterestics of this implementation
+        // for now. We perform all operations up to 128 bytes (which is enough for the current
+        // purposes although this constant can be increased). Any overflow bytes are going to be lost.
+        //
+        // A BigInt is represented as a little endian byte array of size 128 bytes. All
+        // arithmetic operations are performed in base 0x100 (256). The algorithms used
+        // are simply the common primary school techniques for doing operations in base 10.
+        //
+
+        internal sealed class BigInt
```

Comment:
Hmm, it looks like BigInteger should already be available in both the NetStandard.Library and the TargetingPack references.  I think you're just missing a using in the file (System.Numerics).